bitnami/spark/templates/init-configmap.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.initScripts }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name:  {{ printf "%s-init-scripts" (include "common.names.fullname" .) }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+data:
+  {{- include "common.tplvalues.render" ( dict "value" .Values.initScripts "context" $ ) | nindent 4 }}
```

Comment:
In this case, the indentation should be 2 since it is at the "first level"
```suggestion
{{- include "common.tplvalues.render" ( dict "value" .Values.initScripts "context" $ ) | nindent 2 }}
```

See the difference when
1) rendered as it is right now: `helm template . --set initScripts=patata -s templates/init-configmap.yaml`
```yaml
# Source: spark/templates/init-configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name:  release-name-spark-init-scripts
  namespace: "default"
  labels:
    app.kubernetes.io/name: spark
    helm.sh/chart: spark-6.2.0
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/managed-by: Helm
data:
    patata
```

2) rendered using the above suggestion: `helm template . --set initScripts=patata -s templates/init-configmap.yaml`
```yaml
# Source: spark/templates/init-configmap.yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name:  release-name-spark-init-scripts
  namespace: "default"
  labels:
    app.kubernetes.io/name: spark
    helm.sh/chart: spark-6.2.0
    app.kubernetes.io/instance: release-name
    app.kubernetes.io/managed-by: Helm
data:
  patata
```